### PR TITLE
Adds automatic, seasonally appropriate messages from Houston

### PR DESCRIPTION
**Pull request by ElianCodes**
## Changes

- Depends on https://github.com/withastro/cli-kit/pull/29
- Adds a new `data/seasonal.ts` file that implements seasonally appropriate messages and decorations
- Automates Houston's decorations (hats, ties) based on the local date / time of year

## Testing

Tested locally and regressions are covered by the existing tests!

## Docs

No docs

### CODE REVIEW

File: packages/create-astro/src/data/festive.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { random } from '@astrojs/cli-kit/utils';
+
+export default function getFestiveHouston(fancy?: boolean) {
+	const date = new Date();
+	if (date.getMonth() === 11) {
+		// Christmas season
+		return {
+			clothes: {
+				hat: random(['🎁', '🎄', '🌲']),
+				tie: '🧣'
```
Comment (natemoo-re) [inline packages/create-astro/src/data/festive.ts:10]: Technically these should all respect the `fancy` option, which is a good sign that we don't actually need to do that logic in this function. We can do it in the `actions/context.ts` file after we call `getFestiveHouston`/`getSeasonalHouston`.

**Review by natemoo-re — COMMENTED** [review]
This is really fun! I'm glad we won't have to manually manage these messages anymore.

A few suggested refactors to organize things a bit better.

File: packages/create-astro/src/data/seasonal.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { random } from '@astrojs/cli-kit/utils';
+
+export default function getSeasonalHouston(fancy?: boolean) {
+	const date = new Date();
+	if ((date.getMonth() === 11 && date.getDate() === 21) || (date.getMonth() === 0 && date.getDate() === 1)) {
```
Comment (ElianCodes) [inline packages/create-astro/src/data/seasonal.ts:5]:
```suggestion
	if (date.getMonth() === 0 && date.getDate() === 1) {
```

**Review by natemoo-re — APPROVED** [review]
Looks great, thanks for implementing this!

**Review by MoustaphaDev — APPROVED** [review]

**Pull request merged**